config/jobs/kubernetes/sig-windows/OWNERS

```diff
@@ -0,0 +1,19 @@
+reviewers:
+- ddebroy
+- ibabou
+- jayunit100
```

Comment:
These folks are not active anymore, If we move it to the other location we will have a more up to date set of maintainers via https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/sig-windows/OWNERS

Reply:
I've removed [kubernetes-sigs/sig-windows/OWNERS](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/sig-windows/OWNERS)